cmd/meroxa/global/client.go

```diff
@@ -118,7 +118,7 @@ func GetUserToken() (accessToken, refreshToken string, err error) {
 	return accessToken, refreshToken, nil
 }
 
-func NewClient() (*meroxa.Client, error) {
+func NewClient() (meroxa.Client, error) {
```

Comment:
if `meroxa.Client` is not longer a pointer, how come we could reuse it if we haven't returned it?

Reply:
We just talked about this one in a call, so please ignore me.